src/Form.js

```diff
@@ -1,46 +1,51 @@
 import React, {Component} from 'react'
-import { FaPlayCircle, FaListOl, FaHourglassStart } from 'react-icons/fa';
-import { Link } from 'react-router-dom';
+import {FaPlayCircle, FaListOl, FaHourglassStart} from 'react-icons/fa';
+import {Link} from 'react-router-dom';
+import {settings} from "./constants";
 
 class Form extends Component {
-   initialState = {
-     name: '',
-   }
- 
-   state = this.initialState
-   handleChange = event => {
-  
-    this.setState({
-      name: event.target.value
-    })
-  }
-  submitForm = () => {
-    this.props.handleSubmit(this.state)
-    this.setState(this.initialState)
-  }
-
-  render() {
-    const { name } = this.state;
-    return (
-      <form className = "userInput">
-        <label htmlFor="Name" className="inputText"></label>
-        <input
-          type="text"
-          name="Name"
-          id="Name"
-          value={name}
-          onChange={this.handleChange} 
-          placeholder = "Name"
-          className = "inputBox"/> 
-          <big></big> <br />
-          <Link to="/gameplay">
-               <FaPlayCircle className="gameplay-button" onClick = {this.submitForm} />
-         </Link>
-       
-         
-      </form>
-    );
-  }
- }
-  
-  export default Form;
\ No newline at end of file
+    initialState = {
+        name: settings["playerName"],
+    }
+
+    state = this.initialState
+    handleChange = event => {
+
+        this.setState({
+            name: event.target.value
+        })
+
+    }
+    submitForm = () => {
```

Comment:
Good work on the semi-persistent naming, in case the player does not type in their name

Reply:
That is the special case, however the main feature is that if the user modifies their name, it will be saved locally so that when they go back and forth between gameplay and the home page, their name will persist.